cli-client/synthesizer.py

```diff
@@ -182,8 +172,6 @@ def parse_command_line() -> Options:
 
 
 if __name__ == '__main__':
-    print("This module is deprecated.")
     # Setup minimal logger and run example.
```

Comment:
maybe print a warning that this is a test module and not ready yet for real use?

Reply:
Sure I will do it